schemas/nl.basjes/yauaa_context/jsonschema/1-0-4

```diff
@@ -0,0 +1,234 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for an entity context generated by the YAUAA enrichment after parsing the user agent",
```

Comment:
Entity context or context entity?

Reply:
"context entity" feels more right to me. I'd probably just go for "entity" though.

Reply:
I changed it back to just 'context'. I think the word entity confuses things in this case. This schema does not describe an entity, but rather just a grab bag of things that could be deduced from the useragent.